crates/ethereum-rpc/src/lib.rs

```diff
@@ -554,16 +545,15 @@ fn register_rpc_methods<C: sov_modules_api::Context, Da: DaService>(
                 BlockNumberOrTag::Latest => convert_u256_to_u64(evm.block_number(&mut working_set)?),
                 _ => {
                     return Err(to_jsonrpsee_error_object(
```

Comment:
EthApiError::Unsupported("Earliest, pending, safe and finalized are not supported for debug_traceBlockByNumber").into()